pkg/apis/networking/types.go

```diff
@@ -235,12 +235,14 @@ type IngressList struct {
 
 // IngressSpec describes the Ingress the user wishes to exist.
 type IngressSpec struct {
-	// A default backend capable of servicing requests that don't match any
-	// rule. At least one of 'backend' or 'rules' must be specified. This field
+	// The default backend capable of servicing requests that don't match any
+	// rule. At least one of 'defaultBackend' or 'rules' must be specified. This field
 	// is optional to allow the loadbalancer controller or defaulting logic to
-	// specify a global default.
+	// specify a global default. This type is a rename for the spec.Backend for clarity
+	// and will replace the Backend type entirely in a future release.
+	// This type is GA for 1.16
```

Comment:
drop "This type is GA for 1.16", that applies to the external v1 type and is implicit in the version